drone-bom/pom.xml

```diff
@@ -80,6 +80,13 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <!-- Arquillian Drone extension -->
```

Comment:
Same as for Sauce Labs - this part will be merged, so there will be two extensions